core/trino-main/src/main/java/io/trino/operator/annotations/ImplementationDependency.java

```diff
@@ -139,7 +139,7 @@ public static ImplementationDependency createDependency(Annotation annotation, S
                         type);
             }
 
-            throw new IllegalArgumentException("Unsupported annotation " + annotation.getClass().getSimpleName());
+            throw new IllegalArgumentException("Unsupported annotation " + annotation);
```

Comment:
While the check recommends using `annotationType()` instead of `getClass()`, i think showing toString here is even better.